code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -140,6 +139,49 @@
 
 //Spooky special loot
 
+//Rod of Asclepius
+/obj/item/rod_of_asclepius
+	name = "Rod of Asclepius"
+	desc = "A wooden rod about the size of your forearm with a snake carved around it, winding it's way up the sides of the rod. Something about it seems to inspire in you the responsibilty and duty to help others."
+	icon = 'icons/obj/lavaland/artefacts.dmi'
+	icon_state = "asclepius_dormant"
+	var/activated = FALSE
+
+/obj/item/rod_of_asclepius/attack_self(mob/user)
+	if(!activated)
+		if(iscarbon(user))
+			var/mob/living/carbon/itemUser = user
+			to_chat(itemUser, "<span class='notice'>The wooden snake that was carved into the rod seems to suddenly come alive and begins to slither down your arm! The compulsion to help others grows abnormally strong...</span>")
+			if(do_after(itemUser, 40, target = src))
```

Comment:
target needs to be user for the progress bar

Reply:
Sure? If someone disarms the staff it'll still complete in that case, won't it?

Reply:
If that's true it will only be a problem on the ending do_after.

I haven't looked at it in a while but I think target is almost exclusively to set the progress bar and has no bearing on the innate checks of the proc.